app/src/main/java/com/islamey/hamza/wazaker/ui/HomeFragment/HomeFragment.kt

```diff
@@ -41,7 +42,9 @@ class HomeFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        hijriViewModel.getHijriDate(getCurrentDate())
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            hijriViewModel.getHijriDate(getCurrentDate())
+        }
```

Comment:
could we add a comment with reason to do this.

Reply:
as this function is not working with SDK below O

Reply:
share me a link for that.

Reply:
[stackOverFlow link](https://stackoverflow.com/questions/47480732/what-is-the-purpose-of-the-condition-if-build-version-sdk-int-build-version/51381567#51381567)